src/index.html

```diff
@@ -7,6 +7,200 @@
     <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#">
+        <img class="header--logo" src="images/logo.png" alt="moyo logo">
```

Comment:
```suggestion
        <img class="header__logo" src="images/logo.png" alt="moyo logo">
```
это элемент хедера, а не модификатор